lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -190,16 +192,23 @@ def _check_workspace_permissions(self, wsid):
                     f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {wsid}."
                 )
 
-    def _check_workspace_permissions_list(self, wsids):
-        perms = self.sdkmr.get_workspace_auth().can_write_list(wsids)
-        bad_ws = [key for key in perms.keys() if perms[key] is False]
-        if bad_ws:
-            self.logger.debug(
-                f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {bad_ws}."
-            )
-            raise PermissionError(
-                f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {bad_ws}."
+    def _check_workspace_permissions_list(self, wsids) -> None:
+        """
+        Raise exception if you don't have permissions to run a job in any of these workspaces
+        """
+        unique_not_none_wsids = [wsid for wsid in list(set(wsids)) if wsid]
```

Comment:
I don't think you need the `list` before the `set`, `set` is iterable

Reply:
removed list(set()) in favor of set()